guides/standalone-presence.md

```diff
@@ -0,0 +1,57 @@
+# Presence Without Logging
+
+You can use the Disco Log Presence feature independently of logging. This is
+especially useful for apps running on a single node, in which case presence can
+serve as a simple way to display application status.
+
+To set up Presence, first make sure you have disabled the default DiscoLog logger:
+
+```elixir
+config :disco_log,
+  enable: false
+```
+
+Then, start a `DiscoLog.Presence` worker under your app's supervision tree. It's
+probably a good idea to only start it in the `prod` environment:
+
+```elixir
+defmodule MyApp.Application do
+  use Application
+
+  @impl true
+  def start(_type, _args) do
+    children = [
+      # Presence should start after workers required by the HTTP client (e.g. Finch pool)
+      ...
+      presence(),
+      ...
+      MyAppWeb.Endpoint
+    ]
+
+    opts = [strategy: :one_for_one, name: MyAPp.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+
+  if Mix.env() in [:prod] do
```

Comment:
While this work when building local release.
In general `Mix.env()` is not available and it's preferable to use a config instead

Reply:
That's 100% fair for the runtime! At compile-time it's fairly common to use `Mix.env/0` to conditionally define something ([like a dev route](https://hexdocs.pm/swoosh/Swoosh.html#module-mailbox-preview-in-the-browser)). I like this approach because it makes it really hard to accidentally make it a runtime check – the release will simply not start if `Mix.env` call somehow survives compilation and sneaks into runtime